includes/class-wc-stripe-webhook-handler.php

```
@@ -1603,6 +1603,54 @@ public function process_checkout_session( object $notification ): void {
+
+		$message = 'checkout.session.expired' === $notification->type ? __( 'The checkout session has expired.', 'woocommerce-gateway-stripe' ) : __( 'The async payment for this checkout session has failed.', 'woocommerce-gateway-stripe' );
+
+		$status_update         = [];
+		$status_update['from'] = $order->get_status();
+		$status_update['to']   = OrderStatus::FAILED;
+		$order->update_status( OrderStatus::FAILED, $message );
+
+		$this->send_failed_order_email( $order->get_id(), $status_update );
+
+		do_action( 'wc_gateway_stripe_process_webhook_payment_error', $order, $notification );
+	}
```

PHPMD (2.15.0)

[error] The method process_checkout_session_failure is not named in camelCase. (undefined)

(CamelCaseMethodName)